layers/conv_layers.py

```diff
@@ -10,6 +10,17 @@
 from . import OPERATOR
 
 LOG = logging.getLogger("convolution_layers :")
+
+
+# Whether to implement grouped convolution using the native `keras.layers.Conv2D` class with groups !=1 argument.
+# This implementation is supported only with tflite version >= 2.9.
+# If set to `False`, the grouped convolution is built using regular conv per group then concatenated as a workaround
+# to support older version of tflite.
+# Using the native keras implementation results in a simplified tflite graph and supposed to run faster.
+# See https://github.com/MPolaris/onnx2tflite/issues/19 for more details.
+USE_NATIVE_GROUP_CONV = False
```

Comment:
I wasn't sure if it's the best place for this kind of flags. Users will only see this option if they dig down into the code. wdyt?

Reply:
I have no ideal for that, but this way is suitable so far. 
Maybe I will move flag to [entry function](https://github.com/MPolaris/onnx2tflite/blob/main/converter.py#L8) in the future, and I'm also a little confused about how to organize the code effectively.
Thanks for your work!